internal/hardware/validator_test.go

```diff
@@ -220,6 +220,19 @@ var _ = Describe("Disk eligibility", func() {
 		Expect(eligible).ToNot(BeEmpty())
 	})
 
+	It("Check that iSCSI is not eligible when its holder is Multipath.", func() {
```

Comment:
can you add some other cases?
- iSCSI and is eligible
- ISCSI support / not-support on different OCP versions

Reply:
Isn't that what you mean? - https://github.com/openshift/assisted-service/blob/master/internal/hardware/validator_test.go#L163-L186 ?

Reply:
yes but in multipath context 
for example:
- disk is not multipath but one of it's holders is 
- validate  that multipath not supported on all version "4.10","4.12","4.31" 
- validate the returned reason `iSCSIWithMultipathHolder`

we want to cover as much uses cases that can happen in the added if statement

Reply:
> disk is not multipath but one of it's holders is

If the disk itself isn't multipath but its holder is, and we're talking about iSCSI, isn't that what I already tested?

Reply:
yes, I just gave an examples on different thing to test.

Reply:
> validate that multipath not supported on all version "4.10","4.12","4.31"
AFAIK, iSCSI support was introduced only after version 4.15, so the test should fail with a "wrongDriveTypeTemplate" error, similar to the "Check iSCSI on older version is not eligible" test. Do you still want me to test on these specific versions with multipath? Isn't it sufficient to test on iSCSI alone?
In addition, could you please explain why these three specific versions?

> validate the returned reason iSCSIWithMultipathHolder
Done

Reply:
well I'm not sure about the support for it, you can test whatever version you think should be /not be supported
this was just an example